src/main/scala/org/scalameter/Measurement.scala

```diff
@@ -1,15 +1,19 @@
 package org.scalameter
 
+import org.scalameter.picklers.Pickler
 import scala.collection.Seq
 
 @SerialVersionUID(-2541697615491239986L)
-case class Measurement(value: Double, params: Parameters, data: MeasurementData, units: String) {
-  def complete: Seq[Double] = data.complete
+case class Measurement[T: Pickler](value: T, params: Parameters,
```

Comment:
Agreed - using picklers looks like a simpler solution here, consistent with what we were doing before.